github/git_commits_test.go

```diff
@@ -123,6 +127,172 @@ func TestGitService_CreateSignedCommit(t *testing.T) {
 		t.Errorf("Git.CreateCommit returned %+v, want %+v", commit, want)
 	}
 }
+func TestGitService_CreateSignedCommitWithInvalidParams(t *testing.T) {
+	client, _, _, teardown := setup()
+	defer teardown()
+
+	input := &Commit{
+		SigningKey: &openpgp.Entity{},
+	}
+
+	_, _, err := client.Git.CreateCommit(context.Background(), "o", "r", input)
+	if err == nil {
+		t.Errorf("Expected error to be returned because invalid params was passed")
+	}
+}
+
+func TestGitService_CreateSignedCommitWithKey(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+	s := strings.NewReader(testKey)
+	keyring, err := openpgp.ReadArmoredKeyRing(s)
+
+	date, _ := time.Parse("Mon Jan 02 15:04:05 2006 -0700", "Thu May 04 00:03:43 2017 +0200")
+	author := CommitAuthor{
+		Name:  String("go-github"),
+		Email: String("go-github@github.com"),
+		Date:  &date,
+	}
+	input := &Commit{
+		Message:    String("m"),
```

Comment:
Can you please change `"m"` to `"Commit message."`, as it makes line 169 much easier to read below. Thanks.